AR:
  ar_marker_size: 0.022 #ARマーカー一辺. 黒い外枠含む[m]
  ar_type: DICT_4X4_250 #AR markerの種類(https://docs.opencv.org/3.4/d9/d6a/group__aruco.html#gac84398a9ed9dd01306592dd616c2c975)
  map: # IDは連番で、左下のID=0となるよう配置する. gridは最大1000点まで
    range: [[0, 26],[0, 1]]
    density: [0.026, 1]
  id_offset: 0